dask/dataframe/tseries/resample.py

```diff
@@ -4,12 +4,13 @@
 
 from ...base import tokenize
 from ...highlevelgraph import HighLevelGraph
-from ...utils import derived_from
+from ...utils import _deprecated, derived_from
 from .. import methods
 from .._compat import PANDAS_GT_140
 from ..core import DataFrame, Series
 
 
+@_deprecated()
```

Comment:
One small suggestion about adding the version of `dask` after which `getnanos` was deprecated. This will make the deprecation message `getnanos was deprecated after version 2022.02.0...` instead of just `getnanos is deprecated...`, which will both provide users with a bit more info as well as make it easier for maintainers to see exactly when `getnanos` was deprecated (otherwise we'll need to go look at the changelog to see what release included this PR)

```suggestion
@_deprecated(after_version="2022.02.0")
```